notebooks/py_scripts/12_Spike_Burst.py

```diff
@@ -0,0 +1,101 @@
+# ---
```

Comment:
Name of notebook should maybe be something like Spikesorting_Burst_Merge_Curation